test/unit/BPool/BPool_ExitPool.tree

```diff
@@ -0,0 +1,22 @@
+BPool::ExitPool
+├── when reentrancy lock is set
+│   └── it should revert
+├── when pool is not finalized
+│   └── it should revert
+├── when total supply is zero
+│   └── it should revert // division by zero
+├── when pool amount in is too small
+│   └── it should revert
+├── when balance of pool in any token is zero
+│   └── it should revert
+├── when returned amount of a token is less than minAmountsOut
```

Comment:
when calculated amount of any token is less than min?

Reply:
we can do an overall consistency run later and try to get all trees to match a standard 👌